packages/menu/src/Menu.tsx

```diff
@@ -88,46 +89,152 @@ function Menu({
   ...rest
 }: MenuProps) {
   const refs: Array<HTMLElement> = [];
+  const titleArr: Array<string> = [];
+
   const hasSetInitialFocus = useRef(false);
+  const hasSetInitialOpen = useRef(false);
+  // @ts-ignore: using this setState to trigger a subMenu close after animation
+  const [closed, setClosed] = useState(false); // eslint-disable-line
```

Comment:
So rather than adding ts/eslint ignores, I think there's two better ways to write this:
```js
const [ , setClosed] = useState(false);
```
OR
```js
const {1: setClosed} = useState(false);
```

Reply:
👍